src/Containers/WeatherToday/weatherToday.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import {WeatherCard} from "./components/WeatherCard/WeatherCard";
+import {SearchBar} from "./components/searchBar/searchBar";
+import {PreloaderComponent} from "../../Component/preloader/preloader";
+
+export const WeatherToday = (): JSX.Element => {
+    const [cityTitleValue, setCityTitleValue] = useState<string>('')
+    const [cityTitle, setCityTitle] = useState<string>('')
+    const [sunrise, setSunrise] = useState<string>('')
+    const [sunset, setSunset] = useState<string>('')
+    const [weatherState, setWeatherState] = useState<object>({
+        cityTitle: '',
+        humidity: '',
+        feelsLike: 0,
+        temperature: 0,
+        pressure: 0,
+})
+    const [isLoading, setIsLoading] = useState<boolean>(false)
+    const apiKEY = '362efd6a69303a0f5662a19ce25e5e89'
+    let filterTimeout: NodeJS.Timeout;
+
+    useEffect(() => {
+        setIsLoading(true)
+        getCurrentPosition()
+    }, [])
+
+    const doCityFilter = (query: string) => {
+        clearTimeout(filterTimeout)
+        if (!query) return setCityTitleValue('')
+
+        filterTimeout = setTimeout(() => {
+            console.log('====>', query)
+            return getWeatherToday(query).then((data: any): any => setCityTitleValue(data))
+        }, 1500)
+    }
+
+    function getCurrentPosition(): void {
+        const geolocation: Geolocation = navigator.geolocation
+        geolocation.getCurrentPosition(positionHandler, errorPositionHandler, {
+            enableHighAccuracy: true
+        })
+    }
+
+    function positionHandler(position: GeolocationPosition): void {
```

Comment:
some function have typisation of return value and some don't. Either add it everywhere or remove it everywhere.